packages/components/tests/dummy/app/templates/components/form/radio-card.hbs

```diff
@@ -603,7 +603,7 @@
         <span class="dummy-text-small">{{capitalize state}}</span>
         <br />
         <div mock-state-value={{state}} mock-state-selector="label">
-          <Hds::Form::RadioCard {{on "change" this.onChange}} @disabled={{(eq state "disabled")}} as |R|>
+          <Hds::Form::RadioCard {{on "change" this.onChange}} @disabled={{eq state "disabled"}} as |R|>
```

Comment:
@didoo do you know if this actually works? I know I've always included `()` around ember truth helpers, so I wouldn't want us to document something that doesn't work.

Reply:
yes, it's OK: https://cs.github.com/?scope=org%3Ahashicorp&scopeName=hashicorp&q=%3D%7B%7Beq

Reply:
you can see that it works here too: https://hds-components-git-new-docs-website-spike-html-b2436e-hashicorp.vercel.app/components/form/radio-card#showcase (see the disabled state)

<img width="1069" alt="screenshot_1990" src="https://user-images.githubusercontent.com/686239/197870387-9220b518-5a96-481b-b643-02d31c92dc92.png">

Reply:
nice! thanks for the context